### PR TITLE
US003 - view capability for each job role

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/org/kainos/ea/dao/JobRoleDao.java
Comment (leeservice) [inline src/main/java/org/kainos/ea/dao/JobRoleDao.java:1]: We talked about the fact we we're including ID as we need to pass this for capability - all good

File: src/main/java/org/kainos/ea/model/JobRoleGetRequest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package org.kainos.ea.model;
+
+public class JobRoleGetRequest {
```
Comment (ccurran01) [inline src/main/java/org/kainos/ea/model/JobRoleGetRequest.java:3]: It looks like this is the model that we are using as the response. Typically these would be called Response Object. A better name for this class might be 

```suggestion
public class JobRoleResponse {
```

File: src/main/java/org/kainos/ea/model/Capability.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package org.kainos.ea.model;
+
+public class Capability {
```
Comment (ccurran01) [inline src/main/java/org/kainos/ea/model/Capability.java:3]: Doesn't look like this is used anywhere

**Review by leeservice — APPROVED** [review]
This all looks good ryan, thanks! No comments.

**Review by ccurran01 — COMMENTED** [review]
Looks good @ryanbeatt. Few small suggestions. 

Would it be possible to extend the integration or controller tests to check the capability name that is retrieved is what we expected?

File: src/test/java/org/kainos/ea/serviceTests/JobRoleServiceTest.java [hunk 3]
```diff
@@ -44,10 +42,17 @@ void getAllJobRoles_shouldThrowFailedToGetJobRoleException_whenDatabaseConnector
...
+        when(jobRoleDao.getAllJobRoles(conn)).thenReturn(jobRoleResponseList);
+        List<JobRoleResponse> result = jobRoleService.getAllJobRoles();
+        assertEquals(jobRoleResponseList, result);
+
+        // Test capability is displayed
+        JobRoleResponse jobRoleResponse = new JobRoleResponse(1, "Technology Leader", "Engineering");
+        result.add(jobRoleResponse);
+        String expected = "Engineering";
+        String actual = result.get(0).getCapability();
+        assertEquals(expected, actual);
```
Comment (ccurran01) [inline src/test/java/org/kainos/ea/serviceTests/JobRoleServiceTest.java:56]: The coverage from the controller test above might be enough. Not sure that these assertions are adding much value

**Review by ccurran01 — APPROVED** [review]

**Pull request merged**